src/components/InputTimeInterval.vue

```diff
@@ -38,13 +38,24 @@ div
             :disabled="mode == 'range' && (invalidDaterange || emptyDaterange || daterangeTooLong)",
             @click="valueChanged"
           ) Update
+      
+    div(style="float:right;text-align:right" v-if="showUpdate && mode=='last_duration'")
```

Comment:
Avoid float, use flexbox instead. You can use these helper classes: https://getbootstrap.com/docs/4.0/utilities/flex/

Reply:
Thank you for the suggestions😊